File: D-DWM-PG4.9#C配置资料包V6.8/蓝牙安卓手机APP/PG4.6&4.9蓝牙调试助手APP源码/maui-android-0ed0494d7166876cce0b7d853d37dacf61d1821c/src/Maui_pg/Uuids/Pg_Service_Uuids.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Maui_pg.Uuids
{
    public class Pg_Service_Uuids
    {
        public static Guid Pg_Nordic_uart_service_uuid { get; private set; } = new Guid("6e400001-b5a3-f393-e0a9-e50e24dcca9e");
        public static Guid Pg_Nordic_uart_service_rx_character_uuid { get; private set; } = new Guid("6e400002-b5a3-f393-e0a9-e50e24dcca9e");
        public static Guid Pg_Nordic_uart_service_tx_character_uuid { get; private set; } = new Guid("6e400003-b5a3-f393-e0a9-e50e24dcca9e");
    }
}

File: lib/features/inspection/data/services/pg_service_uuids.dart
/// Nordic UART Service UUIDs used by the PG4.9 BLE module.
///
/// Service:  6e400001-b5a3-f393-e0a9-e50e24dcca9e
/// RX char:  6e400002-b5a3-f393-e0a9-e50e24dcca9e  (write to device)
/// TX char:  6e400003-b5a3-f393-e0a9-e50e24dcca9e  (notify from device)
class PgServiceUuids {
  PgServiceUuids._();

  static const String nordicUartService =
      '6e400001-b5a3-f393-e0a9-e50e24dcca9e';

  /// RX characteristic – used to WRITE data to the device.
  static const String nordicUartRx = '6e400002-b5a3-f393-e0a9-e50e24dcca9e';

  /// TX characteristic – used to receive NOTIFY data from the device.
  static const String nordicUartTx = '6e400003-b5a3-f393-e0a9-e50e24dcca9e';
}
